release/.env-template

```diff
@@ -1,24 +1,18 @@
 # To release locally, copy this file to a .env and fill in all values
+
 GITHUB_TOKEN=
 GITHUB_OWNER=
 GITHUB_REPO=metabase
 
-SLACK_BOT_TOKEN=
-SLACK_CHANNEL_RELEASE=
-
 DOCKERHUB_REPO=metabase
 DOCKERHUB_OWNER=
-DOCKERHUB_STAGING_REPO=mb-staging
 DOCKERHUB_RELEASE_USERNAME=
 DOCKERHUB_RELEASE_TOKEN=
 
-AWS_S3_RELEASE_ACCESS_KEY_ID=
-AWS_S3_RELEASE_SECRET_ACCESS_KEY=
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
```

Comment:
Can you explain why you renamed these? And we should remember to update them in the 1password notes